space_packet_parser/xtcedef.py

```diff
@@ -1472,6 +1478,16 @@ def _get_raw_value(self, packet: Packet) -> Union[int, float]:
         """
         raise NotImplementedError()
 
+    @staticmethod
+    def twos_complement(val: int, bit_width: int) -> int:
```

Comment:
This seems like a "helper" function to me, kind of like `_extract_bits()`. Do we want to (a) make it private, and (b) just add it to the bottom of the file as an extra module-level function rather than sticking it onto a class?

Reply:
Agreed. It's only used in our numeric data encodings so I left it on the class. I usually leave static helper methods as public because they can't hurt anything related to the associated class instances but if it's not used elsewhere I can see why it's cleaner to make it private.